cmd/libs/go2idl/conversion-gen/generators/conversion.go

```diff
@@ -685,9 +685,21 @@ func (g *genConversion) doStruct(inType, outType *types.Type, sw *generator.Snip
 			// this field has "genconversion=false" comment to ignore it.
 			continue
 		}
+		t := m.Type
+		if t.Kind == types.Alias {
+			copied := *t.Underlying
```

Comment:
Maybe.  I know we hadn't hit this use case.

On Mon, Jun 13, 2016 at 2:50 AM, Wojciech Tyczynski <
notifications@github.com> wrote:

> In cmd/libs/go2idl/conversion-gen/generators/conversion.go
> <https://github.com/kubernetes/kubernetes/pull/27258#discussion_r66745795>
> :
>
> > @@ -685,9 +685,21 @@ func (g *genConversion) doStruct(inType, outType *types.Type, sw *generator.Snip
> >  			// this field has "genconversion=false" comment to ignore it.
> >  			continue
> >  		}
> > +		t := m.Type
> > +		if t.Kind == types.Alias {
> > +			copied := *t.Underlying
>
> Do we want to support multiple layers of aliasing? I mean if the
> "underlying" is again alias to something?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/kubernetes/kubernetes/pull/27258/files/cfb87dc5632ffef3fba7c0ac49aab7f67b96e6b3#r66745795>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/ABG_p4NKlIrd3wOcnQgwoEwctp8OfgR3ks5qLP3EgaJpZM4Iz54c>
> .
>